1ejer/index.css

```
@@ -0,0 +1,111 @@
+.container{
```

Mejor ponerle algún nombre específico de la página.
```suggestion
.ejer1-container {
```